modules/mobile/app/controllers/mobile/v0/claims_and_appeals_controller.rb

```diff
@@ -13,10 +13,16 @@ module Mobile
   module V0
     class ClaimsAndAppealsController < ApplicationController
       include IgnoreNotFound
+      before_action do
```

Comment:
I'm not sure I understand. that authorization check should be done for all endpoints. Are you thinking we set a proxy variable in the before_action and set it just once instead of having a `if` statement in each action?

Reply:
What I'm saying is that if the flag gets turned on, all endpoints within this controller will authorize for lighthouse access but only one endpoint uses lighthouse. So if someone is getting an appeal from evss, they'll be checked for lighthouse access and not evss access. My suspicion is that this won't matter on prod because all endpoints will be switched before the flag is turned on, and it probably doesn't matter in staging because our test users will have to have access to the same resources in order for this transition to work.

Reply:
oh yeah, that's a good point. you're right that it functionality probably won't effect anything but better safe than sorry. fixed.